android/src/main/java/com/swmansion/reanimated/ReanimatedModule.java

```diff
@@ -64,13 +107,10 @@ public void willDispatchViewUpdates(final UIManagerModule uiManager) {
     final ArrayList<UIThreadOperation> operations = mOperations;
     mOperations = new ArrayList<>();
     uiManager.addUIBlock(
-        new UIBlock() {
-          @Override
-          public void execute(NativeViewHierarchyManager nativeViewHierarchyManager) {
-            NodesManager nodesManager = getNodesManager();
-            for (UIThreadOperation operation : operations) {
-              operation.execute(nodesManager);
-            }
+        nativeViewHierarchyManager -> {
+          NodesManager nodesManager = getNodesManager();
+          for (UIThreadOperation operation : operations) {
+            operation.execute(nodesManager);
```

Comment:
is it backward compatiblie?

Reply:
Fixed in [2efade7](https://github.com/software-mansion/react-native-reanimated/pull/5734/commits/2efade7e68c277b2ab8ca61c31814a910dce2703)